python/dgl/backend/chainer/utils.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+from chainer.backend import get_array_module as _get_array_module
+try:
+    import cupy
```

Comment:
Would this affect the requirements of DGL?

Reply:
Yes; Chainer could install with or without CuPy.